### PR TITLE
add labeling github pull requests

### PR DESCRIPTION
**Pull request by marusinm**
Just copied labeling from issues and create the same for pull requests.

### CODE REVIEW

**Comment by lachmanfrantisek** [comment]
@TomasTomecek  could you help with the zuul?

```yml
  - name: Install git
    dnf:
      name:
      - git-core
      - python36
    become: true
```

Is there some easy way to get the output?

I found this one from [here](https://softwarefactory-project.io/logs/22/122/2d564eea2c4e5a5db82ba9921bd88d0cf505f7fc/check/pre-commit/017f4d8/job-output.json.gz):
```
Failed to download packages: Status code: 503 for https://mirrors.fedoraproject.org/metalink?repo=fedora-30&arch=x86_64
```

**Comment by lachmanfrantisek** [comment]
recheck

File: tests/integration/test_github.py [hunk 1]
```diff
@@ -210,6 +210,12 @@ def test_issue_info(self):
         assert issue_info.title.startswith("add basic readme")
         assert issue_info.status == IssueStatus.closed
 
+    def test_issue_labels(self):
+        self.colin_project.add_issue_labels(issue_id=2, labels=["test_lb1", "test_lb2"])
```
Comment (lachmanfrantisek) [inline tests/integration/test_github.py]: Is it possible to also test the label-number change to be sure that we change something? (We can have these labels already there.)

File: tests/integration/test_github.py [hunk 2]
```diff
@@ -250,6 +256,12 @@ def test_update_pr_info(self):
         assert pr_info.title == "new"
         assert pr_info.description == "new description"
 
+    def test_pr_labels(self):
+        self.colin_project.add_pr_labels(pr_id=4, labels=["test_lb1", "test_lb2"])
+        labels = self.colin_project.get_pr_labels(pr_id=4)
+        assert labels[0].name == "test_lb1"
+        assert labels[1].name == "test_lb2"
```
Comment (lachmanfrantisek) [inline tests/integration/test_github.py]: Same as above.

File: tests/integration/test_github.py (hunk 1 above)
Comment (marusinm) [reply tests/integration/test_github.py]: >Is it possible to also test the label-number change to be sure that we change something? 

I guess you mean labels count on Issue and PR. I added it now in the second commit.

**Pull request merged**